packages/xchain-aggregator/src/types.ts

```diff
@@ -128,13 +128,26 @@ type SwapHistory = {
   swaps: SwapResume[]
 }
 
+type ApproveParams = {
```

Comment:
Export this types in the `index.ts` file

Reply:
Ooops, I didn't see it
[fixed here](https://github.com/xchainjs/xchainjs-lib/pull/1243/commits/05874a29b709bf58441387cbfd08ee38edd31c6b)